admin/src/components/ProductList/SettingLink.js

```diff
@@ -1,37 +1,98 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Grid, GridItem } from '@strapi/design-system/Grid';
 import { Box } from '@strapi/design-system/Box';
 import { Link } from '@strapi/design-system/Link';
 import { Typography } from '@strapi/design-system/Typography';
 import ArrowRight from '@strapi/icons/ArrowRight';
+import pluginPkg from '../../../../package.json';
 
 const SettingLink = () => {
+  const [isNewVersionAvailable, setIsNewVersionAvailable] = useState(false);
+
+  useEffect(() => {
+    (async () => {
+      // call github api to get the latest version of the plugin
+      const responseGithub = await fetch(
```

Comment:
Same code is repeating twice. can we make it as utils function?

Reply:
sure